.vscode/launch.json

```diff
@@ -1,7 +1,4 @@
 {
```

Comment:
Should add `.vscode/` to the `.gitignore` file so this metadata doesn't end up in the repo?

Reply:
Sounds good. Ignored.